.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: CI for sbt-riffraff-artifact
+
+on:
+  pull_request:
+  # Manual invocation.
+  workflow_dispatch:
+  push:
+    branches:
+      - $default-branch
```

Comment:
This should be `main` here, `$default-branch` works within the `.github` repo as some magic templating happens there ✨ .

Reply:
Unfortunately we still have master on this repo. We have card to change it though. I'll set it to master for now.